public/components/marketing-stats/3.html

```diff
@@ -0,0 +1,38 @@
+<section role="region" aria-label="Statistics" class="px-4 py-16 mx-auto flex flex-col items-center justify-center">
+    
+    <div class="text-center md:w-1/2 lg:w-1/2 w-full mx-auto">
+      <h1 class="font-extrabold text-3xl"> Most Prominent In e-commerce </h1>
+      <p class="mt-4 text-gray-500 mx-auto"> Lorem ipsum is a placeholder text commonly used to demonstrate the visual form of a document or a typeface without relying on meaningful content. 
+      visual form of a document or a typeface without relying on meaningful content.
+      </p>
+    </div>
+    <hr  class="w-full mt-10"/>
```

Comment:
Not a fan of this.

Reply:
you mean I should delete the <hr /> tag ?